python/src/equistore/operations/abs.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module to find the absolute values of a :py:class:`TensorMap`, returning a new
+:py:class:`TensorMap`.
+"""
+from ..block import TensorBlock
+from ..tensor import TensorMap
+from . import _dispatch
+
+
+def abs(A: TensorMap) -> TensorMap:
+    r"""
+    Return a new :py:class:`TensorMap` with the same metadata as A and absolute
+    values of ``A``.
+
+    .. math::
+        \nabla(\vert A \vert) = (A/\vert A \vert)*\nabla A
+
+    :param A: input :py:class:`TensorMap` whose absolute values are needed.
+
+    :return: New :py:class:`TensorMap` with the same metadata as ``A`` and
+        absolute values of ``A``.
+    """
```

Comment:
```suggestion
    r"""
    Return a new :py:class:`TensorMap` with the same metadata as A and absolute
    values of ``A``.
    
    .. math::
        A \rightarrow = \vert A \vert

    If gradients are present in ``A``:
    
    .. math::
        \nabla(A) \rightarrow \nabla(\vert A \vert) = (A/\vert A \vert)*\nabla A

    :param A: the input :py:class:`TensorMap`.

    :return: a new :py:class:`TensorMap` with the same metadata as ``A`` and
        absolute values of ``A``.
    """
```

Reply:
nice idea but we never use the -> in the others docstring